nautobot/extras/fields.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
```

Comment:
Since we only use one base here is seems?
```suggestion
from django.db.models import ForeignKey
```